enhancements/olm/olm-managed-operator-metrics.md

```diff
@@ -0,0 +1,145 @@
+---
+title: olm-integration-with-openshift-monitoring
+authors:
+  - "@awgreene"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2019-12-11
+last-updated: 2019-12-11
+status: implementable
+---
+
+# olm-integration-with-openshift-monitoring
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary and Motivation
+
+The primary purpose of this enhancement is to propose a series of changes that would enable a set of Red Hat approved operators to report metrics to [Telemeter](https://github.com/openshift/telemeter).
```

Comment:
are we only concerned with allowing the operators to report metrics, or are we also intending to allow the operand's to do so?  allowing operand's to report metrics will be significantly more complicated, particularly for global operators (the operand's namespace needs to be labeled for metric scraping also).